test/FunctionalTests/Microsoft.OData.Core.Tests/SelectedPropertiesNodeTests.cs

```diff
@@ -63,8 +63,16 @@ public SelectedPropertiesNodeTests()
             this.defaultContainer.AddEntitySet("Cities", cityType);
             this.defaultContainer.AddEntitySet("Districts", districtType);
 
+            EdmComplexType airportType = new EdmComplexType("TestModel", "Airport");

```

Comment:
>airportType [](start = 27, length = 11)

should we add "airportType" into edm model? maybe i miss that.

Reply:
This and regionalAirportType seemed to be implicitly added to the model, but I made them explicit.

---
In reply to: [242373747](https://github.com/OData/odata.net/pull/1359#discussion_r242373747) [](ancestors = 242373747)